sandy/core/cov.py

```diff
@@ -2301,21 +2352,28 @@ def triu_matrix(matrix, kind='upper'):
     return CategoryCov(pd.DataFrame(values, index=index, columns=columns))
 
 
-def sample_distribution(dim, nsmp, seed=None, pdf='normal'):
+def sample_distribution(dim, nsmp, mean=0, seed=None, pdf='normal'):

```

Comment:
This mean and the mean in the method `sampling` are always different or they had to be same?

Reply:
I am not sure we should even have `mean` here.